src/commands/schedule.ts

```diff
@@ -0,0 +1,40 @@
+import type { Command } from "./type";
+import { Message } from "discord.js";
+import { scheduleJob } from "node-schedule";
+
+const days: Record<string, number> = {
+  Sunday: 0,
+  Monday: 1,
+  Tuesday: 2,
+  Wednesday: 3,
+  Thursday: 4,
+  Friday: 5,
+  Saturday: 6,
+};
+
+export const schedule: Command = {
+  name: "schedule",
+  description: "Schedule reoccuring messages",
+  async execute(message: Message, args: string[]): Promise<void> {
+    const [day, time, post] = args;
```

Comment:
`const [day, time, ...post] = args;` will allow for the rest of the message to be posted